tests/log/host/host.c

```diff
@@ -19,6 +21,12 @@ int main(int argc, const char* argv[])
         return 1;
     }
 
+#ifdef CODE_COVERAGE
+    // Disable this test as it writes log to file system
+    // which is override for code coverage purposes
+    return SKIP_RETURN_CODE;
```

Comment:
Can you double check which point does the test fail? Using SKIP_RETURN_CODE with code coverage is only beneficial if some enclave code can be run (so they can contribute to the code coverage data). If we have to return at this point, we should just disable the test.

Reply:
tests/log has been disabled as a whole so I will remove these code changes. As for the point of failure, this test would output and save text to a file on the file system, and since we override filesystem for code coverage, it fails to find the program's output.